src/main/scala/net/ruippeixotog/scalascraper/browser/HtmlUnitBrowser.scala

```diff
@@ -123,6 +123,11 @@ object HtmlUnitBrowser {
     def tagName = underlying.getTagName
     def parent = Option(underlying.getParentNode).collect { case elem: DomElement => HtmlUnitElement(elem) }
     def children = underlying.getChildElements.map(HtmlUnitElement)
+    def siblings = {
+      val previousSiblings = Stream.iterate(underlying.getPreviousElementSibling)(_.getPreviousElementSibling).takeWhile(_ != null)
```

Comment:
Nice usage of `Stream.iterate` :) Don't you prefer to write `Stream.iterate(underlying)(_.getPreviousElementSibling).tail.takeWhile(_ != null)`? If not, please break these statements in two lines, as they are a bit long.